Sources/BSON/ExtendedJSONDecoder.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+/// `ExtendedJSONDecoder` facilitates the decoding of ExtendedJSON into `Decodable` values.
+public class ExtendedJSONDecoder {
+    /// Initialize an `ExtendedJSONDecoder`.
+    public init() {
+        fatalError("unimplemented")
+    }
+
+    /// Decodes an instance of the requested type `T` from the provided extended JSON data.
```

Comment:
I think a `- SeeAlso:` with the following [link](https://docs.mongodb.com/manual/reference/mongodb-extended-json/) would be useful here.

Reply:
added